tests/network/connectivity/utils.py

```diff
@@ -31,7 +33,14 @@ def create_running_vm(
         node_selector=node_selector,
         cloud_init_data=compose_cloud_init_data_dict(
             network_data={
-                "ethernets": {f"eth{i + 1}": {"addresses": [f"10.200.{i}.{end_ip_octet}/24"]} for i in range(0, 3)}
+                "ethernets": {
+                    f"eth{i + 1}": {
+                        "addresses": [
+                            f"{IPV4_ADDRESS_SUBNET_PREFIX_VMI}.{random_octet_ipv4_address[i]}.{end_ip_octet}/24"
+                        ]
+                    }
+                    for i in range(0, MAX_NUM_OF_SECONDARY_ETHERNET_INTERFACES - 1)
```

Comment:
There's no meaning in using this constant here. If you know you need to have 3 iterations - range the iterations from 0 to 3 (just like before this change).